doc/modules/preprocessing.rst

```diff
@@ -517,6 +517,13 @@ features, one has to explicitly set ``categories``. For example,
     >>> enc.transform([['female', 'from Asia', 'uses Chrome']]).toarray()
     array([[ 1.,  0.,  0.,  1.,  0.,  0.,  1.,  0.,  0.,  0.]])
 
+If there is a possibility that the training data might have missing categorical
+features, it is especially important to explicitly set ``categories`` manually.
```

Comment:
Why? If they're not present in training data, then you can't exactly model them... I'd prefer a statement that made `'ignore'` first-class, because in many cases transforming to produce a 0-variance feature is not very helpful.

Reply:
Yes, that makes sense. I tried to reword it to more favor 'ignore', see the last commit.

Reply:
Are you fine with the rewording?